preprocessors/openstreetmap/main.py

```diff
@@ -54,26 +65,26 @@ def get_map_data():
 
     if validated is not None:
         return validated
-
+    time_stamp = int(get_timestamp())
+    name = "ca.mcgill.a11y.image.preprocessor.openstreetmap"
+    request_uuid = content["request_uuid"]
     # Check if this request is for an openstreetmap
     if 'coordinates' not in content and 'placeID' not in content:
-        logging.info("Not map content. Skipping...")
-        return "", 204
+        LOGGER.info("Not map content. Skipping...")
+        return jsonify(""), 204
 
     # Build OpenStreetMap request
     coords = get_coordinates(content)
     if coords is None:
-        error = 'Unable to find Latitude/Longitude'
-        logging.error(error)
-        return jsonify(error), 400
+        error = 'Unable to find API key'
+        LOGGER.error(error)
+        return jsonify(""), 400
```

Comment:
If the API key is not loaded, this is not a client (400) error.

Reply:
@samueljohnsegun148 for a list of errors and what they mean, I recommend looking at: 
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
You can pick and choose the error # you deem is the correct one!

Reply:
@emmanuelwilson, I think I've fixed this already. If you look at the logging message, it is actually about a missing coordinate( latitude/longitude), which should be obviously a client error, but I stand to be corrected.

Reply:
You check if coordinates were not specified above. Please specify if/when `get_coordinates` will return `None` here.

Reply:
@samueljohnsegun148 when will this condition actually be met?

Reply:
So then what does this whole branch of checking if `coords` is `None` actually do?